## The reproducible analysis for the manuscript of the human oral metagenomically assembly

[Over 50000 metagenomically assembled draft genomes for the human oral microbiom
reveal new taxa](https://www.biorxiv.org/content/10.1101/820365v1)

### figure
**Figure 1.** 3,589 oral SGBs assembled from 4154 (3346 new sequence) meta-analyzed oral-wide metagenomes. 
[figure 1c,d](figure/figure1.unknownness.r)

**Figure 2.** The expanded genome set substantially increases the mappability of oral metagenomes. 
[figure 2](figure/figure2_mapping/figure2_mapping_rate.R)

**Figure 3.** Phylogeny of representative oral SGBs.
[figure 3a,b](figure/figure3_phylogeny/figure3ab.phylogeny.r)
[figure 3c](figure/figure3_phylogeny/figure3c.plot_tree.sh)

**Figure 4.** A new candidatus family is found within the Acholeplasmataceae Order.
[figure 4](figure/figure4_newfamily/figure4b_ani_heatmap.r)

**Figure 5.** Geographical distribution of oral SGBs and strains.
[figure 5a,b](figure/figure5_geography/figure5_geograhphy.R)
figure 5c,d[TBD]

**Figure 6.** Disease markers according to the oral genomes.
TDB

**Figure 7.** A comprehensive mapping of the function repertoire of the human oral microbiome.
TDB

### Supplementary Figure
**Figure S1.** Summary of assembly quality, SGBs distribution across 7 studies.
[figure S1](figure/figureS1_assembly_summary/figureS1_summary.R)

**Figure S2.** Complete set of rRNAs and tRNAs in MAGs.
[figure S2](figure/figureS2_RNA/figureS2_RNA_summary.R)

**Figure S3.** Taxonomic classification of reads which unmapped to all genomes in oral genome set.
[figure S3](figure/figureS3_kraken2/figS3_human_ratio.r)

**Figure S4.** Phylogenetic tree and COG functional annotation from all MAGs in the new candidatus family.
[figure S4](figure/figureS4_COGs_new_family/figS4_new_COGs.R)

**Figure S5.** The pangenome genetic diversity.
TBD

**Figure S6.** Heatmap of number of SGBs at family level who share homologous to gut microbiome enzyme coding drug metabolite or healthy related function.
TBD